Scripts/s4ap/utils/s4ap_generic_utils.py

```
@@ -15,17 +22,87 @@ def trigger_autosave(*_) -> bool:
-            CommonBasicNotification(
+            S4APUtils.show_basic_notification(
                 'A problem occured while saving S4AP Data',
                 0
-            ).show()
+            )
             CommonExceptionHandler.log_exception(ModInfo.get_identity(), 'An exception occurred while autosaving.',
                                                  exception=ex)
             return False
+
+    @staticmethod
+    def load_instance(self, instance_type: Types, instance_id: int):
```

Ruff (0.14.7)

Unused static method argument: `self`

(ARG004)